src/duration/extractMinMaxInUnits.js

```diff
@@ -1,5 +1,5 @@
 import durationUnits from './durationUnits';
-import splitSecsToUnits from './splitSecsToUnits';
+import splitSecsToUnits from '../date/splitSecsToUnits';
```

Comment:
Lemme do it ;)
I'll merge this and fix.
We'll use index.js but a project that does not need many of the utils can still reference /lib/object/objFunc